example/e2e/detoxTest.spec.js

```diff
@@ -65,11 +65,7 @@ describe('e2e tests', () => {
       await nextMonthArrow.tap();
       await userDismissesCompactDatePicker();
     } else {
-      const calendarHorizontalScrollView = element(
-        by
-          .type('android.widget.ScrollView')
-          .withAncestor(by.type('android.widget.DatePicker')),
-      );
+      const calendarHorizontalScrollView = getDatePickerAndroid();
```

Comment:
Are you sure you do the right thing here? I see that the intent is **to scroll inside a scrollview**, so why did you replace it with the date picker?

Reply:
Maybe you intended:

```
.withAncestor(by.id('dateTimePicker'))
```

Reply:
Good catch.  Will fix it.